ci/hack/rosa_hcp_setup.sh

```diff
@@ -75,69 +72,73 @@ parse_args() {
     done
 }
 
-init() {
-    # Retrieve AWS Credential file from Bitwarden
-    open_bitwarden_session
-    get_aws_credentials
-    get_rosa_token
-}
-
 prechecks() {
-    if [[ -z "${PREFIX_NAME:-}" ]]; then
-      printf "[ERROR] PREFIX_NAME is not set\n\n" >&2
-      usage
-      exit 1
+    # if rosa is not login, exit
+    if ! rosa whoami; then
+        printf "[ERROR] rosa is not login\n\n" >&2
+        usage
+        exit 1
     fi
-    if [[ -z "${AWS_REGION:-}" ]]; then
-      printf "[ERROR] AWS region is not set\n\n" >&2
-      usage
-      exit 1
+
+    if [[ -z "${AWS_ACCESS_KEY_ID:-}" ]]; then
+        printf "[ERROR] AWS_ACCESS_KEY_ID variable is not exported\n\n" >&2
```

Comment:
You use `is not set` or `is not exported` when vars are not set. Keep the wording consistent or users will be confused.

Reply:
fixed